src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -403,7 +404,7 @@ static internal InvalidOperationException NoConnectionString()
             return InvalidOperation(Res.GetString(Res.ADP_NoConnectionString));
         }
 
-        static internal Exception MethodNotImplemented(string methodName)
+        static internal Exception MethodNotImplemented([CallerMemberName] string methodName = "")
```

Comment:
So callers can call it like `MethodNotImplemented();` instead of `MethodNotImplemented("Caller");` and the compiler will automatically fill in `"Caller"`.